test/framework/github.py

```diff
@@ -114,6 +114,25 @@ def test_read(self):
         except (IOError, OSError):
             pass
 
+    def test_list_prs(self):
+        """Test fetch_easyconfigs_from_pr function."""
+        if self.github_token is None:
+            print "Skipping test_list_prs, no GitHub token available?"
+            return
+
+        parameters = {'state': 'closed', 'sort': 'created', 'direction': 'asc', 'per_page': 1}
+
+        init_config(build_options={'pr_target_account': GITHUB_USER,
+                                   'pr_target_repo': GITHUB_REPO})
+
+        expected = "PR #1: a pr"
+
+        try:
+            output = gh.list_prs(parameters)
```

Comment:
Given all the parsing magic we have, I would test `--list-prs` directly via a test in `test/framework/options.py`?